docs/docs/applications/index.md

```diff
@@ -8,5 +8,34 @@ links:
     next:
     guides:
     related:
+        - applications/add-application
+        - applications/list-applications
+        - applications/view-application
+        - applications/edit-application
+        - applications/deploy-application
+        - applications/deploy-application-using-incoming-hook
+        - applications/integrate-application-deployments-to-ci-workflows
+        - applications/link-server-to-application
     featured:
 ---
+
+## Getting Started with Applications
+
+Applications in Devopness represent your codebase that can be deployed and managed across your infrastructure environments.
+
+### Basic Operations
+
+- [Add an Application](/docs/applications/add-application) - Create and configure a new application
+- [List Applications](/docs/applications/list-applications) - View all applications in your environment
+- [View Application Details](/docs/applications/view-application) - Access detailed information about a specific application
+- [Edit an Application](/docs/applications/edit-application) - Update application settings and configuration
+
```

Comment:
Do we really need all this extra links here?
Aren't those links already accessible after being added to the `related` attribute?